spec/models/place_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+describe Place do
+  it 'can query for its image' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.